continuous_integration/travis_install.sh

```diff
@@ -0,0 +1,134 @@
+  else
+    wget "http://repo.continuum.io/miniconda/Miniconda3-$MINICONDA_VERSION-$MINICONDA_OS.sh" -O miniconda.sh;
+  fi
+
+  bash miniconda.sh -b -p "$HOME/miniconda";
+  export PATH="$HOME/miniconda/bin:$PATH";
+  hash -r;
+  conda config --set always_yes yes --set changeps1 no;
+  conda update -q conda;
+
+  # Useful for debugging any issues with conda
+  conda info -a;
+
+  # Test environments for different Qt bindings
+  if [ "$USE_QT_API" = "PyQt5" ]; then
+    #sudo apt-get install -q "^libxcb.*" libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev
+    #sudo apt-get install -q libxcb-sync0-dev libxcb-render-util0 libxcb-image0  libxcb-xfixes0 libxcb-randr0 libxcb-keysyms1
+    #sudo cp- /usr/lib/libxcb-render-util.so.0 /usr/lib/x86_64-linux-gnu/libxcb-render-util.so.0
+    
+    sudo apt-get install -q "^libxcb.*" libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev
+    #sudo ln -sf /usr/lib/x86_64-linux-gnu/libxcb-render-util.so.0 /usr/lib/libxcb-render-util.so.0 
+
+    conda config --add channels dsdale24;
+    conda create -q -n test-environment python=$PY_VERSION sphinx sip qt5 pyqt5;
+    ldd "$HOME/miniconda/envs/test-environment/lib/qt5/plugins/platforms/libqxcb.so"
+
+    # libxcb-atom1-dev libxcb-event1-dev libxcb-icccm1-dev
+    # ldd /home/goanpeca/anaconda/envs/test-environment/lib/qt5/plugins/platforms/libqxcb.so
+    #	
+
+  elif [ "$USE_QT_API" = "PyQt4" ]; then
+    conda create -q -n test-environment python=$PY_VERSION sphinx sip qt pyqt;
+  elif [ "$USE_QT_API" = "PySide" ]; then
+    conda create -q -n test-environment python=$PY_VERSION sphinx sip qt pyside;
+  fi
+}
+
+install_pyside()
+{
+  # Currently support for python2.7 and python 3.4
+  # http://stackoverflow.com/questions/24489588/how-can-i-install-pyside-on-travis
+  sudo apt-get install libqt4-dev;
+  pip install --upgrade pip;
+  pip install -U sphinx;
+  pip install PySide --find-links "$WHEELHOUSE_URI";  
+  # Travis CI servers use virtualenvs, so we need to finish the install by the following
+  POSTINSTALL=$(find ~/virtualenv/ -type f -name "pyside_postinstall.py";)
+  python $POSTINSTALL -install;
+}
+
+install_qt4()
+{
+  # Install Qt and then update the Matplotlib settings
+  sudo apt-get install -q libqt4-dev pyqt4-dev-tools;
+
+  if [[ $PY_VERSION == 2.7* ]]; then
+    sudo apt-get install python-dev
+    sudo apt-get install -q python-qt4
+  else  
+    sudo apt-get install python3-dev
+  fi
+    # http://stackoverflow.com/a/9716100
+    LIBS=( PyQt4 sip.so )
+
+    VAR=( $(which -a python$PY_VERSION) )
+
+    GET_PYTHON_LIB_CMD="from distutils.sysconfig import get_python_lib; print (get_python_lib())"
+    LIB_VIRTUALENV_PATH=$(python -c "$GET_PYTHON_LIB_CMD")
+    LIB_SYSTEM_PATH=$(${VAR[-1]} -c "$GET_PYTHON_LIB_CMD")
+
+    for LIB in ${LIBS[@]}
+    do
+        sudo ln -sf $LIB_SYSTEM_PATH/$LIB $LIB_VIRTUALENV_PATH/$LIB
+    done
+}
+
+install_qt5()
+{
+  echo "not yet!"
+  sudo add-apt-repository --yes ppa:ubuntu-sdk-team/ppa
```

Comment:
Why is this necessary to use that ppa? Qt5 is already part of the official packages